content/docs/azure/deploy/install-kubeflow.md

```diff
@@ -0,0 +1,132 @@
+
+-   k8s - All Kubernetes resources. Such as Kubeflow packages and add-on packages like fluentd or istio.    
+-   all - Both Azure (WIP) and Kubernetes resources.
+- `${KFAPP} `- the name of a directory where you want Kubeflow configurations to be stored. This directory is created when you runkfctl init. If you want a custom deployment name, specify that name here. The value of this variable becomes the name of your deployment. The value of this variable cannot be greater than 25 characters. It must contain just the directory name, not the full path to the directory. The content of this directory is described in the next section.
+
+### App layout
+
+Your Kubeflow `app` directory contains the following files and directories:
+
+* **app.yaml** - Defines the configuration related to your Kubeflow deployment.
+    * These values are set when you run `kfctl init`.
+    * These values are snapshotted inside `app.yaml` to make your app self contained.
+* **${KFAPP}/ks_app** - A directory that contains the [ksonnet](https://ksonnet.io/) application for Kubeflow.
+    * The directory is created when you run `kfctl generate k8s`.
+    * You can use ksonnet to customize Kubeflow.
+
+
+If you experience any issues running these scripts, see the [troubleshooting guidance](/docs/aws/troubleshooting-azure) for more information.
+
+## Azure Setup
+
+### Login to Azure
+    az login
+### Initial cluster setup for new cluster
+
+Create a resource group:
+    az group create --name <RESOURCE_GROUP_NAME> --location <LOCATION>
+
+Example variables:
+
+- RESOURCE_GROUP_NAME=KubeTest 
+- LOCATION=westus
+
+Create a specifically defined cluster:
+    
+    az aks create -g <RESOURCE_GROUP_NAME> -n <NAME> -s <AGENT_SIZE> -c <AGENT_COUNT> -l <LOCATION> --generate-ssh-keys
+Example variables: 
+
+- NAME=KubeTestCluster 
+- AGENT_SIZE=Standard_D2_v2 
+- AGENT_COUNT=3
+- Use the same resource group and name from the previous step
+
+## Kubeflow Installation
+Run the following commands to set up and deploy Kubeflow. The code below includes an optional command to add the binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the ```kfctl``` binary each time you run it.
+
+1. Create user credentials. You only need to run this command once.
+
+        az aks get-credentials --name <NAME> --resource-group <RG>
+
+1. Run the following commands to download the latest binary from the Kubeflow releases page (link = https://github.com/kubeflow/kubeflow/releases/).
+
+        wget <link to your release>
+
+1. Unpack the tar ball
+
+        tar -xvf kfctl_<release tag>_<platform>.tar.gz
+
+1. Run the following commands to set up and deploy Kubeflow. The code below includes an optional command to add the binary kfctl to your path. If you don’t add the binary to your path, you must use the full path to the `kfctl` binary each time you run it.
+```
+# The following command is optional, to make kfctl binary easier to use.
```

Comment:
The formatting is broken from this point onwards. See the preview:
https://deploy-preview-780--competent-brattain-de2d6d.netlify.com/docs/azure/deploy/install-kubeflow/#the-following-command-is-optional-to-make-kfctl-binary-easier-to-use

To fix, try adding an empty line and indenting the code snippet. See this file for an example:
https://raw.githubusercontent.com/kubeflow/website/master/content/docs/gke/deploy/deploy-cli.md